content/news/2023-02-22-news-2023-02-22-new-r-ladies-website/index.en.md

```diff
@@ -0,0 +1,103 @@
+
+- Multilingual website 
+  - This is not the easiest to maintain with Wordpress
+- Very slow directory
+  - The database was just too slow and heavy, page loading time was horrible
+- Integrate the blog into the website
+  - The blog was a Blogdown site maintained through Github and Netlify
+- Easier long-term maintenance and collaboration
+  - Wordpress would require making a user for every person that wanted to contribute to the site
+  - Changing to something hosted on GitHub would mean easier community help and collaboration
+
+## History
+The work already started in 2019, with Bea Hernandez, Daloha Rodríguez-Molina and Maëlle Salmon.
+The initial work was in making a [blogdown]() website, which utilises [Hugo]() and Rmarkdown integration.
+It was a natural place to start porting our Wordpress site into something more common to use in our community. 
+Additionally, the Wordpress site was requiring more and more maintenance, and in particular our [R-Ladies directory]() was so slow we were getting reports of people not using it because the loading time of the page was so long!
+
+In 2020, I (Athanasia Mowinckel) was on-boarded to the website team, initially to maintain the Wordpress site while the new Hugo site was being built.
+After just a little while, I also started working on the blogdown site. 
+
+In mid-2020 we decided that the webpage was better served as a pure Hugo site, not as blogdown. 
+At that point, it was due to some Hugo features not existing in Blogdown (yet) that we wanted to utilise for the website.
+These were settings for multi-lingual websites we really wanted to take advantage of.
+Furthermore, since only the blog would ever need code execution, having a site build that does not need R means a much faster build, and easier to maintain.
+
+Then Covid hit, and we all felt the stress of that period.
+Development went slow, and things dragged out.
+Thankfully, the Leadership contacted me and asked if I needed help to get the last pieces in place, and that we could set aside a little budget to hire help for the javascript pieces I was struggling with.
+
+We [announced]() the need, and hired Ben Ubah to help me get the last crucial pieces in place. Finally, we were nearing completion!
+
+## Release & new features!
+
+We have now released the new website and are thrilled about how it works so far!
+The theme is well suited to us, and the content is much easier to deal with now that we can collaborate through GitHub.
+It's also made it possible to integrate the website with certain other automatic pipelines, which enables us to have a couple of new features to the website from before!
+
+- [Events page]() with a calendar of R-Ladies events
+  - These are fetched daily from meetup through their API
+- [Direcotry page]() which is actually fast!
+  - Updated and maintained in another private repository with Airtable integration
+- [Blog]() where we welcome contributing posts and cross-posts!
```

Comment:
do we have some contributing guidelines? should we hold off announcing this before we do if we don't?